### ISSUE TITLE
File Browser Theme Styling

### ISSUE THREAD

**Issue by Mark-Agent003** [LMMS/lmms#4070]
The file browser currently doesn't respond to mouse hovering and is overall a bit user-unfriendly. Changing the color or spacing when a mouse hovers over an item makes it easier for users to know what item they are about to click, especially if their mouse is not near the item titles or in the margins. It also makes the interface seem more responsive and user-friendly. I have a style.css file where I've added hover response, click response (the indentation of an item you click will increase), and I've also increased the font size. I've attached a screenshot of the changes in action.

![image](https://user-images.githubusercontent.com/25230510/34329784-ea142488-e8d2-11e7-8f42-a726f9b3ce43.png)

**Comment by Moth-Tolias** [comment]
it'd be nice to have at least the option to use the OS-built-in interface on windows; I'm guessing that isn't a possibility on other platforms, though?

**Comment by tresf** [comment]
> it'd be nice to have at least the option to use the OS-built-in interface on windows; I'm guessing that isn't a possibility on other platforms, though?

@Moth-Tolias, Qt has some native-style integration, but it would make the application inconsistently themed across platforms, so it's not something we're in a rush to support.  If someone wants to spearhead a native theme, we'd consider adding it to the list of native themes, but we would need a proof of concept first.

Here's the code for making Qt's Look and Feel behave close to the native OS's Look and Feel:

```c++
#ifdef LMMS_BUILD_WIN32
    QApplication::setStyle("WindowsVista") || QApplication::setStyle("WindowsXP") || QApplication::setStyle("Windows");
#elif LMMS_BUILD_APPLE
    QApplication::setStyle("Macintosh");
#else
    QApplication::setStyle("GTK") || QApplication::setStyle("Fusion");
#endif
```

> The file browser currently doesn't respond to mouse hovering and is overall a bit user-unfriendly

@Mark-Agent003, I'm not sure what you mean by "Not user friendly" as this is a blanket term, but I do not find the "indent" to be consistent UI with anything file browsers that I'm familiar with.  The "hover" feedback would be a welcome addition.  I would recommend mimicking the behavior of the rest of the software.  Please submit a PR.  👍

**Comment by tresf** [comment]
Closed via 59eba30.  @Mark-Agent003 thanks for the work on this, greatly appreciated!
